modules/schema_validation.lua

```diff
@@ -193,8 +204,15 @@ function module.mt.__index:CompareType(data_elem, schemaElem, isArray, nameofPar
     if(self.schema.interface[interface_name].enum[complex_elem_name][data_elem] ~= nil) then 
       return true
     end 
+    --Enum element can be sended as number, so if it includes to data elements, we will set it as acceptable
```

Comment:
@AByzhynar   I meant if number is less than count of elements in enum, then we assume it belongs to enum